roles/autotested/tasks/produce_results.yml

```diff
@@ -16,10 +16,7 @@
 
 # Extract a name unique to this run, but common across runs of same job
 - name: Attempt to run alternate results2junit
-  shell: >
-    {{ autotest_docker_path }}/results2junit
-        --name {{ peon_metadata.name }}
-        results/default
+  shell: {{ autotest_docker_path }}/results2junit --name {{ peon_metadata.name }} results/default
```

Comment:
This needs to be quoted.  The parser sees the ``{`` at the beginning and tries to convert the whole line from JSON